### ISSUE TITLE
Localization not added for voiceover in the quiz 

### ISSUE THREAD

**Issue by karna1810** [FalconRealityDev/Falcon-CareUpGame#2]
- When the user clicks on a wrong option in the quiz, the voiceover says  **"Hmm, nee, dat klopt niet."**
- This has to be translated to English as **"Hmm, no, that's not right."** and the correct voiceover should be played depending on the language selected

**Comment by karna1810** [comment]
Added Translated Audio and added code to switch audio according to the selected language
